tests/sentry/api/endpoints/test_organization_release_details.py

```diff
@@ -548,6 +550,156 @@ def test_get_prev_and_next_releases_on_sessions_sort_with_query_filter_applied(
         assert response2.data["currentProjectMeta"]["prevReleaseVersion"] == "foobar@1.2.0"
         assert response2.data["currentProjectMeta"]["nextReleaseVersion"] == "foobar@1.0.5"
 
+    def test_get_first_and_last_release_on_date_sort(self):
+        """
+        Test that ensures that the first release and the last release in terms of `date_added` are
+        retrieved correctly
+        """
+        release_1 = Release.objects.create(
+            organization_id=self.organization.id, version="foobar@1.0.0"
+        )
+        release_1.add_project(self.project1)
```

Comment:
jfyi, you can use `self.create_release` here and save a lot of boilerplate

Reply:
Yes you are right, fixed! I will create another PR to reduce boilerplate on the other pagination tests as well. thanks